### PR TITLE
EC-3037 Added icons to generic promo

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
-          [[#icon]]
-              <span class="icon icon-[[.]]"></span>
-          [[/icon]]
+    <div class="content-text-upper">
+      <div class="promo-title-wrap">
+        <div class="promo-title[[#icon]] with-icon[[/icon]][[#tags]] has-tags-after[[/tags]][[#relation]] has-relation-after[[/relation]]">
+          [[#url]]<a href="[[.]]">[[/url]][[title]][[#url]]</a>[[/url]]
         </div>
-
+        [[#icon]]
```
Comment (mirjamverloop) [inline source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache]: Is icon still in use or has it been replaced by the is_3d, is_audio, etc.?

File: source/sass/search/_channels-object-promos-base.scss [hunk 2]
```diff
@@ -615,6 +615,13 @@
...
+    height:            2.7em;
+    -webkit-transform: translate(-0.4em, 0) scale(0.7);
+    transform:         translate(-0.4em, 0) scale(0.7);
+    width:             2.8em;
+  }
```
Comment (mirjamverloop) [inline source/sass/search/_channels-object-promos-base.scss]: I think the icon should be a little bit smaller and there should only be the padding of 1rem between the icon and the right side of the card.

File: source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache (hunk 1 above)
Comment (andyjmaclean) [reply source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache]: It's used by the next/prev cards

File: source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache [hunk 3]
```diff
@@ -10,14 +10,9 @@
         <div class="promo-title[[#icon]] with-icon[[/icon]][[#tags]] has-tags-after[[/tags]][[#relation]] has-relation-after[[/relation]]">
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply source/_patterns/js_template/cho/promotions/channel-object-promo-text-js.mustache]: Done

**Pull request merged**